projects/1.18/assets/italian-delights-add-on-for-farmers-delight/italian_delight/lang/zh_cn.json

```
@@ -1,10 +1,14 @@
 {
+"advancements.awkward.descr": "喝下一杯未完成的'拙'酒",
+  "advancements.awkward.title": "等等...这太尴尬了",
   "advancements.covermeinwine.descr": "制作你的第一份红酒萨拉米！",
   "advancements.covermeinwine.title": "红酒裹身！",
   "advancements.dragon_borne.descr": "喝下龙裔酒",
   "advancements.dragon_borne.title": "你是……龙裔！",
   "advancements.first_pizza.descr": "制作你的第一份披萨！",
   "advancements.first_pizza.title": "披萨时间！",
+  "advancements.xmas_achivment.descr": "制作一份潘多酪！",
```

```suggestion
  "advancements.xmas_achivment.descr": "制作一份潘多洛！",
```
貌似找到的一般是这么音译的
本身是个面板，用`酪`总感觉怪怪的